src/main/java/com/example/music/backend/playlist/domain/Playlist.java

```diff
@@ -0,0 +1,38 @@
+package com.example.music.backend.playlist.domain;
+
+import com.example.music.backend.user.domain.User;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+@Table(name = "playlists")
```

Comment:
use the same name  convention, plrease."VIDEOS" and "playlists" should have the same formatting

Reply:
I only use camelcase